### PR TITLE
Doc correction - Fixed Broken Link

### LINKED ISSUE #35232: Broken link
The link to Clean Architecture is broken: https://8thlight.com/blog/uncle-bob/2012/08/13/the-clean-architecture.html
It redirects to https://8thlight.com/Insights

Suggestion for internal link:
https://learn.microsoft.com/en-us/dotnet/architecture/modern-web-apps-azure/common-web-application-architectures#clean-architecture

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 49346f80-1204-08d7-df5e-3495c2404eca
* Version Independent ID: 9ece45e7-6f8a-8c13-78a0-fb1b95ed3fe1
* Content: [Designing a microservice-oriented application](https://learn.microsoft.com/en-us/dotnet/architecture/microservices/multi-container-microservice-net-applications/microservice-application-design)
* Content Source: [docs/architecture/microservices/multi-container-microservice-net-applications/microservice-application-design.md](https://github.com/dotnet/docs/blob/main/docs/architecture/microservices/multi-container-microservice-net-applications/microservice-application-design.md)
* Product: **dotnet-architecture**
* Technology: **microservices**
* GitHub Login: @jamesmontemagno
* Microsoft Alias: **jamont**

### PR DESCRIPTION
**Pull request by Adi9235**
## Summary

I have fixed the broken link for Clean Architecture and the correct link is added which is 
https://learn.microsoft.com/en-us/dotnet/architecture/modern-web-apps-azure/common-web-application-architectures#clean-architecture

Fixes #35232 

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/architecture/microservices/multi-container-microservice-net-applications/microservice-application-design.md](https://github.com/dotnet/docs/blob/c6e6dd96ef5bec50774217cc59ece7b82bc15971/docs/architecture/microservices/multi-container-microservice-net-applications/microservice-application-design.md) | [Design a microservice-oriented application](https://review.learn.microsoft.com/en-us/dotnet/architecture/microservices/multi-container-microservice-net-applications/microservice-application-design?branch=pr-en-us-35672) |
| [docs/fsharp/language-reference/sequences.md](https://github.com/dotnet/docs/blob/c6e6dd96ef5bec50774217cc59ece7b82bc15971/docs/fsharp/language-reference/sequences.md) | [Sequences](https://review.learn.microsoft.com/en-us/dotnet/fsharp/language-reference/sequences?branch=pr-en-us-35672) |

### CODE REVIEW

**Comment by Adi9235** [comment]
Hey can someone help me in this i.e. how to pass the failed check of OPS status checker I am not able to understand it where to correct.

File: docs/architecture/microservices/multi-container-microservice-net-applications/microservice-application-design.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ There are many architectural patterns used by software architects and developers
 
 - [Domain-Driven Design N-layered](https://devblogs.microsoft.com/cesardelatorre/published-first-alpha-version-of-domain-oriented-n-layered-architecture-v2-0/).
 
-- [Clean Architecture](https://8thlight.com/blog/uncle-bob/2012/08/13/the-clean-architecture.html) (as used with [eShopOnWeb](https://aka.ms/WebAppArchitecture))
+- [Clean Architecture](https://learn.microsoft.com/en-us/dotnet/architecture/modern-web-apps-azure/common-web-application-architectures#clean-architecture) (as used with [eShopOnWeb](https://aka.ms/WebAppArchitecture))
```
Comment (gewarren) [inline docs/architecture/microservices/multi-container-microservice-net-applications/microservice-application-design.md:162]:
```suggestion
- [Clean Architecture](../../modern-web-apps-azure/common-web-application-architectures.md#clean-architecture) (as used with [eShopOnWeb](https://aka.ms/WebAppArchitecture))
```

**Comment by jamesmontemagno** [comment]
Can you revert your changes to the F# docs? 1 PR per fix please ;) <3

**Comment by Adi9235** [comment]
> Can you revert your changes to the F# docs? 1 PR per fix please ;) <3
should I revert the latest commit ? can you help in this for reverting

**Pull request closed without merging**